packages/conf-nanomsg/conf-nanomsg.0/opam

```diff
@@ -1,12 +1,12 @@
 opam-version: "1"
-maintainer: "Rudi Grinberg"
+maintainer: "Vincent Bernardoff"
 authors: ["Martin Sustrik"]
 homepage: "http://nanomsg.org/"
 license: "MIT/X11"
-build: [["pkg-config" "libnanomsg"]]
+build: [["test" "-f" "/usr/include/nanomsg/nn.h"]]
```

Comment:
This doesnt seem like it'll work outside a few Linux distros (what about Homebrew or FreeBSD in /usr/local)? /cc @rgrinberg as well.